tests/Simplify/DictTest.elm

```diff
@@ -365,6 +365,26 @@ a = Dict.fromList [ ( key, v0 ), ( key, v1 ) ]
                             |> Review.Test.whenFixed """module A exposing (..)
 import Dict
 a = Dict.fromList [ ( key, v1 ) ]
+"""
+                        ]
+        , test "should replace Dict.fromList [ ( key, v0 ), thing, ( key, v1 ) ] by Dict.fromList [ thing, ( key, v1 ) ]" <|
```

Comment:
maybe also add the same test for Set?
Otherwise really good refactors and fixes.
I keep missing cases like that so I'll try to test on more examples next time :)